src/main/scala/http4s/extend/instances/package.scala

```diff
@@ -1,6 +1,8 @@
 package http4s.extend
 
 package object instances {
-  object sync extends SyncInstances
-  object errorInvariantMap extends ErrorInvariantMapInstances
+  object eq                 extends EqInstances
```

Comment:
I'm not sure what'd be the best practices here, if do instances by typeclasses or instances by type (i.e `ThrowableCompleteMessage`). I guess it makes more sense to having them by typeclasses but on the other hand you might lose that capability of just bringing the instances of the type that you are using, if that makes sense. Having said that, libraries like cats have a mix of both. 
I don't have a clear suggestion on this but this is just an attempt to acknowledge it. xD

Reply:
it have been bothering me as well. Not sure which way to go though yet

Reply:
I would leave it this way for now, until we have a clearer idea. What do you think ?